crypto/test/src/asn1/test/ASN1IntegerTest.cs

```diff
@@ -60,7 +60,7 @@ public override void PerformTest()
             }
             catch (ArgumentException e)
             {
-                IsEquals("malformed integer", e.Message);
```

Comment:
wouldn't the right way to do this be: `StringAssert.Contains("malformed integer", e.Message);`

Reply:
I agree there's probably a better way to do this, but I was just following what this commit did:

https://github.com/bcgit/bc-csharp/commit/194d05d2d8f4ad5eb2d1d940cb692096f1210485

Reply:
Yeah the entire IsTrue static method also baffles me - I think Bouncycastle is showing some age in it's unit tests at least XD...